.github/workflows/docker-images.yml

```diff
@@ -89,6 +90,7 @@ jobs:
     name: Publish ${{ inputs.lang }} to Docker Hub
     runs-on: ubuntu-latest
     if: inputs.do-push && !github.event.pull_request.head.repo.fork
+    timeout-minutes: 30
```

Comment:
I saw a [runaway job](https://github.com/temporalio/omes/actions/runs/17308616687/job/49144094871?pr=171) that still wasn't done after 2h+